src/Microsoft.Framework.PackageManager/Restore/NuGet/HttpSource.cs

```diff
@@ -276,6 +285,19 @@ private static bool IsFileAlreadyOpen(string filePath)
             return false;
         }
 
+        private string GetCacheFilePath(string cacheKey)
+        {
+            var baseFolderName = RemoveInvalidFileNameChars(ComputeHash(_baseUri));
+            var baseFileName = RemoveInvalidFileNameChars(cacheKey) + ".dat";
+
+#if DNX451
+            var localAppDataFolder = Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData);
+#else
+            var localAppDataFolder = Environment.GetEnvironmentVariable("LocalAppData");
```

Comment:
I am moving existent code to a helper method. Fixing cross-platform issues should be done in a separate thread. I filed an issue for it: https://github.com/aspnet/dnx/issues/1813